backend/queries/events.js

```diff
@@ -288,6 +288,95 @@ const getSingleEventAdmin = async (eId) => {
 //   return await db.any(selectQuery);
 // }
 
+//Get all events data for non-admin dashboards
+const getAllEventsNonadmin = async (usertype, userId) => {
```

Comment:
since this function has pastsQuery and upcomingsQuery that are related to volunteers, I think it's better to let it specific for only volunteers, maybe something like `getAllEventsVolunteer = async (volunteerId)`